SPECS/kernel/kernel-uki.spec

```diff
@@ -65,11 +65,19 @@ install -vdm 700 %{buildroot}/lib/modules/%{kernelver}
 install -vm 600 vmlinuz-uki.efi %{buildroot}/boot/vmlinuz-uki-%{kernelver}.efi
 ln -s /boot/vmlinuz-uki-%{kernelver}.efi %{buildroot}/lib/modules/%{kernelver}/vmlinuz-uki.efi
 
+# install to ESP
+install -vdm 700 %{buildroot}/boot/efi/EFI/Linux
+cp %{buildroot}/boot/vmlinuz-uki-%{kernelver}.efi %{buildroot}/boot/efi/EFI/Linux/vmlinuz-uki-%{kernelver}.efi
```

Comment:
Why copy instead of a symbolic link like what's happening a few lines above?

Reply:
The ESP is a FAT32 partition, so symlinks are not supported

Reply:
Thank you, I've learned something!:)